neptune_optuna/impl/__init__.py

```diff
@@ -273,37 +357,49 @@ def log_study_metadata(study: optuna.Study,
         >>> study = optuna.create_study(direction='maximize')
         ... study.optimize(objective, n_trials=5)
 
-        Log Study metadata to Neptune:
+        Log single and multi-objective Study metadata to Neptune:
         >>> import neptune.new.integrations.optuna as optuna_utils
         ... optuna_utils.log_study_metadata(study, run)
 
+        Or `optionally` pass a list of objective names:
+            Single-objective:
+            ... optuna_utils.log_study_metadata(study, run, target_names=['accuracy'])
+            Multi-objective:
+            ... optuna_utils.log_study_metadata(study, run, target_names=['FLOPS', 'accuracy'])
+
+
     For more information, see `Neptune-Optuna integration docs page`_.
     .. _Neptune Optuna integration docs page:
        https://docs.neptune.ai/integrations-and-supported-tools/hyperparameter-optimization/optuna
     """
     run = run[base_namespace]
 
+    namespaces = _get_namespaces(study, target_names)
+
+    _log_best_trials(run, study, namespaces)
```

Comment:
I don't see such a function (only a method in callback class but it is unrelated). Could you double check that this is ok?